components/ProductPage/index.tsx

```diff
@@ -210,7 +210,7 @@ export default async function CategoryProduct(props: ProductsProps) {
                         )}
                     </div>
 
-                    <div className="col-span-5 relative">
+                    <div className="mt-10 lg:mt-0! lg:col-span-5 relative">
```

Comment:
Does `!` at the end do anything here `lg:mt-0!`?

Reply:
Yeah, we need that, otherwise seems that it doesn't remove the margin on large screens

Reply:
But it usually is inftont - `lg:!mt-0`, that is why I guess it does notthing